### ISSUE TITLE
make html failing with node 12.8.0

### ISSUE THREAD

**Issue by JJ** [perl6/doc#2949]
## The problem

Due apparently to a problem with the highlighter oniguruma, build is failing here:

```
459 verbose node v12.8.0
460 verbose npm  v6.10.2
461 error code ELIFECYCLE
462 error errno 1
463 error oniguruma@6.2.1 install: `node-gyp rebuild`
463 error Exit status 1
464 error Failed at the oniguruma@6.2.1 install script.
464 error This is probably not a problem with npm. There is likely additional logging output above.
465 verbose exit [ 1, true ]
```

## Suggestions

It might be a version-locked problem, or it might be that all the setup has become obsolete. I think @antoniogamiz is ready to substitute it, but meanwhile we need to get this going.

**Comment by JJ** [comment]
This seems to be the error:
```
class Nan::Persistent<v8::Object>’ has no member named ‘IsNearDeath’
```
It's persisting after an npm audit fix and rebuild.

**Comment by JJ** [comment]
Closed with 70fb14ad360e203926bc878bc958ececd8137b6c
